### PR TITLE
adding articles and youtube video

### PR DESCRIPTION
**Pull request by lejlasp**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/documentation/index.md [hunk 1]
```diff
@@ -3,14 +3,19 @@ title: Code documentation
 ---
 ## Code Documentation
 
-Code documentation is a way developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie to get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
+Code documentation is a way in which developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie to get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
```
Comment (rupali317) [inline guide/english/documentation/index.md]: Please change "Code Documentation lets the newbie to get comfortable with..." to "Code Documentation lets the newbie get comfortable with....". Basically just omit the "to".

File: guide/english/documentation/index.md [hunk 2]
```diff
@@ -3,14 +3,19 @@ title: Code documentation
...
 For example, if you happen to pass along your code to an absolute beginner, they should be able to follow along through comments, appropriate variable names and that means structuring the code in a spaced, readable manner. 
 
 It may become an extremely important idea to make a habit out of commenting your functions, loops, and declarations and treating comments as part of the source code, just as regular code should be.
 
 ![An example of commenting in the real world](https://cdn-images-1.medium.com/max/1620/1*Pyxsc7Uixbitv5myywaA_Q.jpeg)
 
+Presentation on code documentation:
+<br/><a href="http://www.youtube.com/watch?feature=player_embedded&v=pubm3spwkog
+" target="_blank"><img src="http://img.youtube.com/vi/pubm3spwkog/0.jpg" 
+alt="IMAGE ALT TEXT HERE" width="240" height="180" border="10" /></a>
```
Comment (rupali317) [inline guide/english/documentation/index.md]: For **alt="IMAGE ALT TEXT HERE"**, the value of alt seems very generic. We should describe what the image is about. When the image fails to load then "IMAGE ALT TEXT HERE" won't be helpful in describing what the image is about.

One possible value  of alt attribute could be "Documentation for Better Code"

**Review by rupali317 — CHANGES_REQUESTED** [review]

File: guide/english/documentation/index.md [hunk 3]
```diff
@@ -3,14 +3,19 @@ title: Code documentation
 ---
 ## Code Documentation
 
-Code documentation is a way developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie to get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
+Code documentation is a way in which developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
```
Comment (thecodingaviator) [inline guide/english/documentation/index.md]:
```suggestion
Code documentation is a way in which developers write their code to create the best versions of their functions possible. Code Documentation lets a newbie get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
```

File: guide/english/documentation/index.md [hunk 4]
```diff
@@ -3,14 +3,19 @@ title: Code documentation
...
 
 For example, if you happen to pass along your code to an absolute beginner, they should be able to follow along through comments, appropriate variable names and that means structuring the code in a spaced, readable manner. 
 
 It may become an extremely important idea to make a habit out of commenting your functions, loops, and declarations and treating comments as part of the source code, just as regular code should be.
 
 ![An example of commenting in the real world](https://cdn-images-1.medium.com/max/1620/1*Pyxsc7Uixbitv5myywaA_Q.jpeg)
 
+Presentation on code documentation:
+<br/><a href="http://www.youtube.com/watch?feature=player_embedded&v=pubm3spwkog
```
Comment (thecodingaviator) [inline guide/english/documentation/index.md]: You don't need the line break here, please also change the link to markdown format

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Check my comments

**Comment by RandellDawson** [comment]
@thecodingaviator This one has gone stale

**Comment by thecodingaviator** [comment]
Hi @lejlasp 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**